modules/ROOT/pages/connected-apps-overview.adoc

```diff
@@ -16,6 +16,25 @@ OAuth (Open Authorization) is an open protocol that provides secure API authoriz
 OAuth can authorize access to resources without revealing user credentials to apps.
 OpenID Connect identifies the end user and obtains information to pass to OAuth 2.0 connected apps.
 
+
+// Sara add some indentation or mark up here please
+
+In order to understand the purpose of Connected Apps, there are several technical concepts and terms to be familiar with, beginning with OAuth 2.0 (commonly referred to as just OAuth) and OpenID Connect (commonly referred to as OICD). Before breaking down these terms, it’s important to understand that these flows are completed in order to give access to Anypoint APIs to only those services and users with the proper permissions. 
+
+OAuth 2.0
+When understanding OAuth 2.0, an authorization protocol, we must first define several key terms to ground our understanding. The first is the client, which refers to the resource that wants the data and could be a computer or server. Next, the authorization server refers to the server that can authorize, and the resource server refers to the system with the data. The access token is the key to get into data in the resource server and the redirect URI is where you go if the user or service authorizes the other service.
```

Comment:
No first person in docs. Additionally, we cannot lead the users through a narrative and tell them what to understand; we simply provide the information. This is a great writing convention for blogs and more casual material, but technical documentation has to be very tightly woven and not tell them what to think; all it can do is state what is.

Reply:
```suggestion
The OAuth 2.0 authorization protocol depends on several elements to function. 

Client::
The resource that requests data. This could be a computer or server.

Authorization server::
The server that can authorize. 

Resource server::
The system that contains the requested data.

Access token:: 
A "key" needed to access data in the resource server. 

Redirect URI::
The destination if the user or service authorizes the other service.
```

Reply:
@sebastiankorol Do you have good answers to these? I'm not sure how to clarify.

Reply:
You can refer to [RFC 6749](https://tools.ietf.org/html/rfc6749) for a precise definition of these terms

Reply:
I tried to put these in my own words based on Seba's resource

Client: The resource that requests data on behalf of a user or service.
Auth server: The server that gives access tokens to the client once it does authorization
Redirect URI: The destination the authorization server sends the user or server once the application is authorized. 

Also feel free to link the link Seba provided

Reply:
Updated this, so Let me know